eventRate.js

```diff
@@ -50,6 +56,7 @@ function startDraw()
 {
   startTime = performance.now();
   eventCount = 0;
+  coalescedEventCount = 0;
```

Comment:
Yeah. I wanted to show the coalesced event rate as well but I forgot. Now looking at your other comment it seems only the rate of the real events matter. So I just removed it.